ts/apps/polaris-cli/src/app/yargs-commands/migrate-command.ts

```diff
@@ -0,0 +1,47 @@
+import { CommandModule } from 'yargs';
+
+import { NX_CLI, PolarisCli } from '../polaris-cli';
+import { RunNpmBinaryTask, Task } from '../tasks';
+import { POLARIS_PKGS } from '../util/packages';
+import { getLatestReleaseVersion, readFromPackageJson as getNrwlPackages, getNxVersion } from '../util/packages-utils';
+import { createYargsCommand } from './command';
+
+const VERSION = 'polarisVersion';
```

Comment:
should this be hardcoded?

Reply:
yes, but it can be something else than `polarisVersion`. `polaris-version` maybe?